internal/file/structure.go

```diff
@@ -25,18 +25,12 @@ const (
 	// This directory contains all the SSL certificates that are specified in Ingress rules.
 	// The name of each file is <namespace>-<secret name>.pem. The content is the concatenated
 	// certificate and key.
-	DefaultSSLDirectory = "/ingress-controller/ssl"
+	DefaultSSLDirectory = "/etc/ingress-controller/ssl"
```

Comment:
I know this is not directly related to this PR but I've been thinking, since you're making a change to the layout here, could you also set the container work directory to /etc/nginx? :)
That'd be super convenient to be able to show the config with just `kubectl exec ... -- cat nginx.conf` instead of the full path.
Just a suggestion.

Reply:
done